sockeye/arguments.py

```diff
@@ -895,6 +895,11 @@ def add_training_args(params):
                               choices=[C.RNN_INIT_ORTHOGONAL, C.RNN_INIT_ORTHOGONAL_STACKED, C.RNN_INIT_DEFAULT],
                               help="Initialization method for RNN parameters. Default: %(default)s.")
 
+    train_params.add_argument('--fix-params',
```

Comment:
I would vote for calling this "fixed-params" or even "fixed-param-names", which is consistent with what MXNet expects.

Reply:
I was tempted to do so as well, but I have the feeling that most of our arguments (if appropriate) refer to actions: --restrict-lexicon, --fill-up, --use-tensorboard, etc.

Reply:
True, but to me `--fix-params` sounds like a boolean flag (fix params if this flag is given) and doesn't encourage me to put a list of names after it).